roles/business.md

```diff
@@ -2,6 +2,22 @@
 
 ## Business Traits
 
+### +Academy
+
+We need someone:
+
+- to prepare and deliver the weekly learning programmes
+- to provide appropriate feedback and coaching, to help academy members to succeed  
+- to use a variety of teaching methods to challenge and engage academy members
+- to be the PO for the greenfield project
+
+We expect someone in the role:
+
+- to be autonomous in leading the academy
+- to be well organised and committed
+- to be accountable for progress and showcases back to the business
```

Comment:
Make a less soggy statement would be something 

* "Is responsible for evolving the academy to be more successful"
* "Is accountable for showcases of progress every x weeks"

What I'm a bit unsure about is showcase what? Progress of improvement of academy members? Or improvement of academy practices itself?